### PR TITLE
Task#3997

### LINKED ISSUE #3997: [Entry] JavaScript Dates: .toLocaleString()
### Reason/inspiration (optional)

We would like a new term entry for JavaScript [Dates](https://www.codecademy.com/resources/docs/javascript/dates) about the `.toLocaleString()` method. The entry should go in a new file in `docs/content/javascript/concepts/dates/terms/toLocaleString/toLocaleString.md`

The entry should:
    - Have an introductory paragraph about what the method is and does.
    - Have a syntax section illustrating the function in a ` ```pseudo ` code block. 
    - An example showing the function in a ` ```js ` code block. 
    - A runnable codebyte example in a ` ```codebyte/javascript` block. 

    
Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by kpoultney**
### Description

I have created a folder under content > javascript > concepts > dates > terms named toLocaleString. In this folder, I have created a file named toLocaleString.md detailing the term entry of the .toLocaleString() method. I used Mozilla MDN docs as a reference guide for my entry. This contribution was done as a project (Contributing to the Codecademy Docs Repository) for the Codecademy course Learn Git and GitHub.

### Issue Solved

Closes #3997 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by Helfese — APPROVED** [review]

File: content/javascript/concepts/dates/terms/toLocaleString/toLocaleString.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+// Using Options
+const options = {
+  weekday: 'long',
+  month: 'long',
+  day: 'numeric',
+  year: 'numeric',
+};
+
+console.log(date.toLocaleString(options))
+// 11/22/1963, 12:30:00 PM
```
Comment (ishg-153) [inline content/javascript/concepts/dates/terms/toLocaleString/toLocaleString.md:45]: Under each of the `console.log` you have given the output in comments. Please remove that.

File: content/javascript/concepts/dates/terms/toLocaleString/toLocaleString.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+```
+
```
Comment (ishg-153) [inline content/javascript/concepts/dates/terms/toLocaleString/toLocaleString.md:47]:
```suggestion

The above code block gives the following output:

```shell
11/22/1963, 5:30:00 PM
1963. 11. 22. 오후 5:30:00
11/22/1963, 5:30:00 PM


```

File: content/javascript/concepts/dates/terms/toLocaleString/toLocaleString.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+---
+
+The **.toLocaleString()** date method returns a string of a given [`Date` object](https://www.codecademy.com/resources/docs/javascript/dates) in the local timezone using the local formatting and language. This translation is provided according to the parameters `locale` and `options`, otherwise the default is set to the browser's location.
+
+## Syntax
+
+```js
+date.toLocaleString(locale, options)
+```
+The parameters `locale` and `options` can be used. `Locale` is represented by a language subtag and a region subtag (i.e. `en-US`). `Options` is an object defining the format for `weekday`, `year`, `month`, `day`, `dayPeriod`, `hour`, `minute`, `second` and `fractionalSecondDigits`. If undefined, the default setting is "numeric".
```
Comment (ishg-153) [inline content/javascript/concepts/dates/terms/toLocaleString/toLocaleString.md:22]:
```suggestion
The parameters `locale` and `options` can be used. `Locale` is represented by a language subtag and a region subtag (i.e. `en-US`). `Options` is an object defining the format for `weekday`, `year`, `month`, `day`, `dayPeriod`, `hour`, `minute`, `second`, and `fractionalSecondDigits`. If undefined, the default setting is "numeric".
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@kpoultney I've suggested a couple of changes. Please make those and let me know once done.

I apologize for my delayed response.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@kpoultney I'm marking previous suggestions as unresolved, please take a look and make the necessary changes. Make sure you push your changes to `Task#3997` branch. No need to create a new PR, we will keep working on this same one.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@kpoultney You still need to address on suggestion marking that as unresolved, please make it.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@kpoultney The change you are referring to is not reflecting as complete hence I am marking it as unresolved. Please take a look and make the suggested change and push it.

**Comment by ishg-153** [comment]
@kpoultney I think you accidentally closed this repository because you deleted your fork of the Codecademy docs repo. Are you still willing to work on this issue?

**Comment by kpoultney** [comment]
No

**Pull request closed without merging**